boards/we/ophelia4ev/doc/index.rst

```diff
@@ -0,0 +1,96 @@
+.. zephyr:board:: ophelia4ev
+
+Overview
+********
+
+.. note::
+   You can find more information about the nRF54L15 SoC on the `nRF54L15 website`_.
+   For the nRF54L15 technical documentation and other resources (such as
+   SoC Datasheet), see the `nRF54L15 documentation`_ page.
+
+The OPHELIA-IV EV board is an evaluation board of the Ophelia-IV radio module.
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Hardware
+********
+
+The Ophelia-IV uses the internal low frequency RC oscillator
+and provides the so called smart antenna connection, that allows
+to choose between the module's integrated PCB antenna and an external
+antenna that can be connected to the available SMA connector.
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Programming and Debugging
+*************************
+
+Applications for the ``ophelia4ev/nrf54l15/cpuapp`` board target can be
+built, flashed, and debugged in the usual way. See
+:ref:`build_an_application` and :ref:`application_run` for more details on
+building and running.
+
+Applications for the ``ophelia4ev/nrf54l15/cpuflpr`` board target need
+to be built using sysbuild to include the ``vpr_launcher`` image for the application core.
+
+Enter the following command to compile ``hello_world`` for the FLPR core:
+
+.. code-block:: console
+
+   west build -p -b ophelia4ev/nrf54l15/cpuflpr --sysbuild
+
+
+Flashing
+========
+
+As an example, this section shows how to build and flash the :zephyr:code-sample:`hello_world`
+application.
+
+.. warning::
+
+   When programming the device, you might get an error similar to the following message::
+
+    ERROR: The operation attempted is unavailable due to readback protection in
+    ERROR: your device. Please use --recover to unlock the device.
+
+   This error occurs when readback protection is enabled.
+   To disable the readback protection, you must *recover* your device.
+
+   Enter the following command to recover the core::
+
+    west flash --recover
+
+   The ``--recover`` command erases the flash memory and then writes a small binary into
+   the recovered flash memory.
+   This binary prevents the readback protection from enabling itself again after a pin
+   reset or power cycle.
```

Comment:
I suggest moving the warning AFTER the actual build instructions -- L88

Reply:
All changes done. Thank you for the feedback!